src/Common/SettingsChanges.h

```diff
@@ -28,6 +28,13 @@ class SettingsChanges : public std::vector<SettingChange>
     bool tryGet(std::string_view name, Field & out_value) const;
     const Field * tryGet(std::string_view name) const;
     Field * tryGet(std::string_view name);
+
+    /// inserts element if doesn't exists and returns true, else just returns false
```

Comment:
Consider to rewrite `else just returns false` into `otherwise returns false`.

Reply:
"else just" is used to emphasize that existing value is not changed - "otherwise" would be ambiguous in this regard